src/styles/components/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  display: grid;
+  padding: 50px 40px;
+  grid-template-columns: repeat(4, 200px);
```

Comment:
Consider using `1fr` instead of a fixed `200px` for the grid-template-columns property to make the grid layout more flexible. Fixed sizes can cause issues on different screen sizes or resolutions. For instance, `grid-template-columns: repeat(4, 1fr);` would allow the columns to take up an equal fraction of the available space.